playbooks/katello_atomic_registry.yml

```diff
@@ -0,0 +1,4 @@
+- hosts: all
```

Comment:
todo: add docs? what is the preferred form of documenting how to use and what variables there are?